### PR TITLE
chore: style improvements to component explorer

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Styling improvements to component explorer.
Screen Shot 2019-08-06 at 2 50 10 PM

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-component-explorer/app/explorer.tsx [hunk 1]
```diff
@@ -233,7 +262,7 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
                                         "managedClasses.explorer_paneTitleContainer"
                                     )}
                                 >
-                                    <Label>Component Explorer</Label>
+                                    <Heading size={HeadingSize._6}>Explore</Heading>
```
Comment (janechu) [inline packages/fast-component-explorer/app/explorer.tsx]: is this the expected title?

File: packages/fast-component-explorer/app/explorer.tsx [hunk 2]
```diff
@@ -653,6 +724,23 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
...
+    private handleDevToolsToggle = (
+        e: React.MouseEvent<HTMLButtonElement>,
+        props: ActionToggleProps
+    ): void => {
+        this.setState({
+            devToolsVisible: !props.selected,
+        });
+    };
+
+    private handleDevToolsItemClick = (e: React.MouseEvent<unknown>): void => {
```
Comment (janechu) [inline packages/fast-component-explorer/app/explorer.tsx]: can you rename this to `handleDevToolsPivotTriggerClick`

File: packages/fast-component-explorer/app/explorer.tsx (hunk 1 above)
Comment (eljefe223) [reply packages/fast-component-explorer/app/explorer.tsx]: Thats whats in the Figma files:
image

Comment (eljefe223) [reply packages/fast-component-explorer/app/explorer.tsx]: Updated to "FAST Explorer" after talking to @bheston

**Pull request merged**